.circleci/config.yml

```diff
@@ -74,7 +74,15 @@ jobs:
           command: yarn test:projects
       - run:
           name: Bundle Statistics
-          command: yarn build:stats
+          command: yarn stats:build
+      - run:
+          name: Save Statistics to DB
+          command: |
+            if [ -n "${STATS_URI}" ]; then
+              yarn stats:save
+            else
+              echo "STATS_URI not set, skipping"
```

Comment:
Seems this should throw, no?  Do we want silent passes when stats aren't being saved for some reason?